src/___new___/components/FelixConfigTable/index.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+const TableCRD = ({ fieldData }) => (
+  <table>
+    <thead>
+      <tr>
+        <th>Attribute</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+      <td>Key</td>
+      <td><code>{fieldData.NameYAML || 'No Default Value'}</code></td>
+      </tr>
+      <tr>
+        <td>Description</td>
+        <td dangerouslySetInnerHTML={{ __html: fieldData.DescriptionHTML }} />
+      </tr>
+      <tr>
+        <td>Schema</td>
+        <td dangerouslySetInnerHTML={{ __html: fieldData.YAMLSchemaHTML }} />
+      </tr>
+      <tr>
+        <td>Default</td>
+        <td>
+                    {fieldData.YAMLDefault === '' ? (
+                      'none'
+                    ) : (
+                      <code>{fieldData.YAMLDefault}</code>
+                    )}
+                  </td>
+      </tr>
+    </tbody>
+  </table>
+);
+
+const TableConfig = ({ fieldData }) => (
+  <table>
+    <thead>
+      <tr>
+        <th>AttributeConfig</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+      <td>Key</td>
+      <td><code>{fieldData.NameConfigFile || 'No Default Value'}</code></td>
+      </tr>
+      <tr>
+        <td>Description</td>
+        <td dangerouslySetInnerHTML={{ __html: fieldData.DescriptionHTML }} />
+      </tr>
+      <tr>
+        <td>Schema</td>
+        <td dangerouslySetInnerHTML={{ __html: fieldData.StringSchemaHTML }} />
+      </tr>
+      <tr>
+        <td>Default</td>
+        <td>
+                    {fieldData.YAMLDefault === '' ? (
+                      'none'
+                    ) : (
+                      <code>{fieldData.YAMLDefault}</code>
+                    )}
+                  </td>
+      </tr>
+    </tbody>
+  </table>
+);
+
+const TableEnv = ({ fieldData }) => (
```

Comment:
For the sake of neatness, you could extract these components into their own files and import them
```
/FelixConfig
  - index.js
  - file.json
  - TableEnv.js
  - TableConfig.js
  - TableCRD.js

Reply:
Done.